### ISSUE TITLE
When Launched from Studio, bunnies have incorrect phenotype

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/natural-selection#76]
Noticed in natural-selection master with sha 7fd382c99c46d3396451d519f4261dcdd56673f6.  Also reproducible in https://phet-dev.colorado.edu/html/natural-selection/1.0.0-dev.38/phet-io/

To reproduce:

1. Run the sim in Studio.
2. Go to the _Lab_ screen.
3. In the "Add Mutations" panel, press the Fur button in the "Dominant" column.
4. Press the "Add a Mate" button.
5. Stop the sim after 1 or more brown bunnies appear, and it will look something like this (note brown bunny at upper-right):

<img width="830" alt="screenshot_307" src="https://user-images.githubusercontent.com/3046552/81884926-74f28b80-9556-11ea-8e4f-d6cef7ddc2c7.png">

6. Press the "Launch" button in Studio.  The sim launches and you'll see something like this:

<img width="833" alt="screenshot_308" src="https://user-images.githubusercontent.com/3046552/81884934-78861280-9556-11ea-9c13-818abe8807b4.png">
In the launched sim, we have the correct number of bunnies, in the correct positions, facing in the correct directions.  But all of the bunnies are white; there are no brown bunnies. So the phenotype (which describes appearance) is definitely wrong.  And phenotype is derived from genotype.
<br><br>
@zepumph FYI in case you have any immediate suggestions.  I haven't dug into this yet.

**Comment by pixelzoom** [comment]
Ah, think I found the problem.  Bunny.phenotype is not currently part of state.

**Comment by pixelzoom** [comment]
Yep, that was the problem, missing phenotype in Bunny state.  Corrected in the above commit.  Closing.
